docs/ism/api.md

```diff
@@ -394,6 +422,14 @@ GET _opendistro/_ism/explain/<index>
 
 ## Get ISM status
 
+Returns the current status of the ISM plugin. We have three different states the ISM plugin can be in:
+
+1. Running
+2. Stopping
+3. Stopped
+
+This is because we have some actions that are considered unsafe to stop in the middle of. When you try to stop ISM it will move into the stopping state
```

Comment:
ISM considers some action unsafe to stop in the middle of, so when you try to stop the plugin, it moves into the `stopping` state.